src/Tribe/Admin/Event_Settings.php

```diff
@@ -369,11 +359,13 @@ public function settings_ui( $admin_page ) {
 			return;
 		}
 
-		include_once Tribe__Main::instance()->plugin_path . 'src/admin-views/tribe-options-general.php';
-		include_once Tribe__Main::instance()->plugin_path . 'src/admin-views/tribe-options-display.php';
+		include_once \Tribe__Main::instance()->plugin_path . 'src/admin-views/tribe-options-general.php';
+		include_once \Tribe__Main::instance()->plugin_path . 'src/admin-views/tribe-options-display.php';
+		include_once tribe( 'tec.main' )->plugin_path . 'src/admin-views/tribe-options-general.php';
+		include_once tribe( 'tec.main' )->plugin_path . 'src/admin-views/tribe-options-display.php';
```

Comment:
Are we including these twice?

Reply:
No, one is coming from common, the other from TEC